orb-attest/src/remote_api.rs

```diff
@@ -120,7 +120,7 @@ fn format_secret(val: &str) -> String {
 }
 
 fn powercycle_security_mcu() -> std::io::Result<()> {
-    Command::new("mcu-util")
+    Command::new("orb-mcu-util")
```

Comment:
should we consider using full bin paths here?
cc @eth44

Reply:
Great catch @karelispanagiotis, yes we want to use full paths everywhere

Reply:
addressed in https://github.com/worldcoin/orb-software/pull/123